api/api/utils/pagination.py

```diff
@@ -31,16 +31,23 @@ def get_paginated_response_schema(self, schema):
         """
 
         field_descriptions = {
-            "result_count": "The total number of items returned by search result.",
-            "page_count": "The total number of pages returned by search result.",
-            "page_size": "The number of items per page.",
-            "page": "The current page number returned in the response.",
+            "result_count": (
+                "The total number of items returned by search result.",
+                10000,
+            ),
+            "page_count": ("The total number of pages returned by search result.", 20),
+            "page_size": ("The number of items per page.", 20),
+            "page": ("The current page number returned in the response.", 1),
         }
         return {
             "type": "object",
             "properties": {
-                field: {"type": "integer", "description": description}
-                for field, description in field_descriptions.items()
+                field: {
+                    "type": "integer",
+                    "description": description,
+                    "example": example,
+                }
+                for field, [description, example] in field_descriptions.items()
```

Comment:
Pedantic suggestion to use tuples for unpacking when the item is a tuple.
```suggestion
                for field, (description, example) in field_descriptions.items()
```

Reply:
Nice, thank you, @dhruvkb !